DOCSP-42363: Add association_ids code examples (#41)
mongodb/docs-mongoid: source/release-notes/mongoid-8.0.txt

association previously loaded into memory, Mongoid 8.0 uses the in-memory collection to satisfy the query
instead of issuing a new query to the database.

For example, the following code calls the ``member_ids`` method and uses the in-memory
collection to retrieve the ``id`` values. The ``id`` of the ``new_member`` is not included
in the output:

.. code-block:: ruby

   ids = band.member_ids
   # ...

   new_member = Member.create(band_id: band.id)

   # Prior to 8.0, the method output includes the newly-created member's id.

>    # As of 8.0, the output does not include the newly-created member's id.

Reviewer: ```suggestion
   # Starting in 8.0, the output does not include the newly-created member's id.
```

   p band.member_ids

You can force an ``<association>_ids`` method to issue a new database query
by resetting the association before calling ``<association>_ids``, as shown
in the following code: 

.. code-block:: ruby

   band.members.reset

   # Queries the database rather than the in-memory collection
   p band.member_ids

You can also issue a new database query by reloading the parent record,

> as shown in the following code:

Reviewer: S: explain where in your code the code should be to reload so that you can retrieve the new id
